<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DLTRelaisForm</class>
 <widget class="QWidget" name="DLTRelaisForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>148</width>
    <height>138</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DLT Relais</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QCheckBox" name="checkBoxRelais1">
     <property name="text">
      <string>State</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="pushButtonRelais1">
     <property name="text">
      <string>Trigger</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="checkBoxRelais2">
     <property name="text">
      <string>State</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pushButtonRelais2">
     <property name="text">
      <string>Trigger</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="checkBoxRelais3">
     <property name="text">
      <string>State</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="pushButtonRelais3">
     <property name="text">
      <string>Trigger</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="checkBoxRelais4">
     <property name="text">
      <string>State</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="pushButtonRelais4">
     <property name="text">
      <string>Trigger</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
